.github/workflows/ci.yaml

```diff
@@ -17,12 +17,15 @@ jobs:
         uses: actions/checkout@v2
       - name: Set up Python
         uses: actions/setup-python@v2
-      - run: python -m pip install flake8 flake8-import-order doc8 sphinx
+      - run: |
+          python -m pip install flake8 flake8-import-order doc8 sphinx \
+            rstcheck[sphinx]
       - run: python -m pip install .
       - run: flake8 .
       - run: doc8 $(git ls-files '*.rst')
       - run: yamllint --strict $(git ls-files '*.yaml' '*.yml')
       - run: python setup.py build_sphinx
+      - run: rstcheck --ignore-directives automodule $(git ls-files '*.rst')
```

Comment:
Can you put it in the linting steps (just before running yamllint on files and building docs)? It would make more sense to me :+1: